hail/python/hailtop/utils/utils.py

```diff
@@ -200,6 +205,14 @@ async def call(self, f, *args, **kwargs):
     def call_nowait(self, f, *args, **kwargs):
         self._queue.put_nowait((f, args, kwargs))
 
+    def shutdown(self):
+        for worker in self.workers:
+            if worker:
```

Comment:
Same question about checking whether the worker is done as above

Reply:
whoops again, same deal.